engine/src/flutter/ci/builders/mac_ios_engine.json

```diff
@@ -248,253 +248,6 @@
                     "sdk_version": "16c5032a"
                 }
             }
-        },
```

Comment:
I wonder if we should keep building these artifacts for a release (even though it doesn't really work) in case someone is using it, and then put the deprecation in the tech blog with instructions for migrating? I bet people have prototypes that "work" locally and I don't want them to be mysteriously busted.